src/Components/test/E2ETest/Tests/StandaloneAppTest.cs

```diff
@@ -108,6 +108,22 @@ public void HasFetchDataPage()
             }
         }
 
+        [Fact]
+        public void IsStarted()
+        {
+            // Read from property
+            var jsExecutor = (IJavaScriptExecutor)Browser;
+
+            var isStarted = jsExecutor.ExecuteScript("return window['__aspnetcore__testing__blazor_wasm__started__'];");
+            if (isStarted is null)
+            {
+                Assert.True(false, "Blazor wasm started value not set");
```

Comment:
Nit, is Assert.True(false) better than just throwing an exception with the error?

Reply:
That makes sense, updating to:
```suggestion
                throw new InvalidOperationException("Blazor wasm started value not set");
```